src/services/lista-nominal/cards.ts

```
@@ -0,0 +1,79 @@
+/**
+ * Builds URL with given parameters
+ * @param baseUrl - Base URL to add parameters to
+ * @param params - Object containing listName, ine, cardType and municipio_id_sus
+ * @returns URL with added query parameters
+ */
+export const buildUrlWithParams = (
+  baseUrl: string,
+  params?: {
+    listName: string,
+    ine?: string,
+    municipio_id_sus: string;
```

Biome (1.9.4)

[error] This property name should be in camelCase.



(lint/style/useNamingConvention)